subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -799,6 +879,9 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_commands,
 		"-f: Force to use this regulatory hint over any other regulatory hints\n"
 		"Note: This may cause regulatory compliance issues, use it at your own risk.",
 		cmd_wifi_reg_domain),
+	SHELL_CMD(ps_timeout, NULL,
+		  "Configure Wi-Fi power save inactivity timer(in ms)",
```

Comment:
Ideally the API can also be called set_ps_inactivitity, but I guess this is already merged upstream, so, lets leave it for now.